src/main/java/org/elastos/hive/vendors/onedrive/OneDriveAuthHelper.java

```diff
@@ -65,11 +65,15 @@ public AuthToken getToken() {
 
 	@Override
 	public CompletableFuture<AuthToken> checkExpired(Callback<AuthToken> callback) {
-		if (token.isExpired())
+		if (token.isExpired()) {
```

Comment:
Just:
if (token.isExpired())
    return redeemToken(callback)

Do not need "{ }" if then statements have one line.

Reply:
Ok, i'll modify it.